scripts/Editor/DllManipulatorEditor.cs

```diff
@@ -8,21 +8,24 @@
 #endif
 using System.IO;
 using System;
+using UnityEditorInternal;
```

Comment:
Required for `AssemblyDefinitionAsset` to find all asmdefs in the project in `RefreshAllKnownAssemblies()`.

Reply:
Removed in a2a212cc2ba011dd8cbe43b0ccc22fd85fa7f742